source/mainnet/net/release-notes/release-notes-lp.rst

```diff
@@ -302,13 +302,17 @@ Desktop Wallet
 |mw-gen1| for iOS
 -----------------
 
-    May 30, 2023
+    August 8, 2023
 
-    In |mw-gen1| for iOS 3.2.0 identity and account creation has been locked in |mw-gen1| for iOS devices. This means that you cannot create new identities or accounts in |mw-gen1| on an iOS device. You can continue to use |mw-gen1|, but if you need to create a new identity or account you must use |mw-gen2|. You can also still recover your wallet from a backup file in |mw-gen1| on an iOS device.
+        |mw-gen1| 3.2.1 ensures support for the upcoming P6 protocol update.
```

Comment:
I thought the P in P6 stood for protocol 6. Assuming that is correct, it reads a bit funny "the upcoming protocol 6 protocol update".

Reply:
Maybe this?

```suggestion
        |mw-gen1| 3.2.1 ensures support for the upcoming protocol version 6.
```